mne/viz/epochs.py

```diff
@@ -442,6 +438,254 @@ def plot_epochs(epochs, epoch_idx=None, picks=None, scalings=None,
     return fig
 
 
+def plot_epochs(epochs, picks=None, scalings=None, n_epochs=20,
+                n_channels=10, fig_title=None, show=True, block=False):
+    """ Visualize single trials.
```

Comment:
we never talk about trials but epochs. I know it's the same but let's be consistent with naming